src/bls12_381/ec/g1.rs

```diff
@@ -35,6 +36,46 @@ impl fmt::Debug for G1Uncompressed {
     }
 }
 
+/// These methods provide fast reading and writing for `G1Affine` points.
+/// Points are guaranteed to be unaffected by a `write`-`read` roundtrip,
+/// but input to `read` is assumed to be correct. No validation is performed
+/// on the raw components, so it is an error to `read` arbitrary data.
+impl G1Affine {
```

Comment:
please add this for G2Affine too, they are kept fully in sync

Reply:
#14